html/css/dark.css

```diff
@@ -1,3 +1,11 @@
+.ie9 img {
+    width: 100%;
+}
+@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+    img {
+        width: 100%;
```

Comment:
Yes, I'm not convinced that this PR is the best way to go about fixing it.

I might remove the CSS changes, and leave the additions of viewbox to the SVGs. I think those are going to be required regardless.